### DISCUSSION TITLE
No More Wi-Fi Calling On T-Mobile US After Updating Device

### DISCUSSION THREAD

**Discussion by blaford** [kyujin-cho/pixel-volte-patch#266] [Carrier-specific Experiences]
After updating my Xperia 5 Mk IV to build 64.1.A.0.996 it seems to have lost wifi calling. Even after total reinstall it doesn't appear to work. 

Oddly enough I can enable it in "Calls & SMS" in settings, but it doesn't show up in notification or work when I disable mobile service.

**Comment by fillwithjoy1** [comment]
Okay, so just wondering you've tried to turn on the toggles with this app I presume?

**Comment by blaford** [comment]
Toggles appear and work just fine. However if I set to airplane mode, turn
on wifi, and try to make a call while connected to my home network... I get
a message that I need to connect to a wifi network.

**Comment by fillwithjoy1** [comment]
Have you made sure that your IMS Status is `Registered` in the app before doing this? (and also check your Wi-Fi calling settings)

**Comment by fillwithjoy1** [comment]
Send screenshot of IMS status in the `*#*#4636#*#*` and see if you can get a logcat at all
